src/Service.Tests/Unittests/ConfigFileWatcherUnitTests.cs

```diff
@@ -72,7 +72,6 @@ HostMode mode
     /// on hot reload.
     /// </summary>
     [TestMethod]
-    [Ignore]
```

Comment:
did you sync with @RubenCerna2079 about this as he was working on [#2109](https://github.com/Azure/data-api-builder/issues/2109) already.

Reply:
Not yet, he was oof today so I just added the fix here, which was small.

Reply:
But will sync tomorrow when he is back :)